src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -12,6 +12,7 @@
 #include "replication/service/raft_repl_service.h"
 #include "replication/repl_dev/raft_repl_dev.h"
 #include "push_data_rpc_generated.h"
+#include "boost/lexical_cast.hpp"
```

Comment:
nit: It is not mandatory, but common practice we decided to follow was to define the 
STL defs, followed by external libraries both indicate by <> (instead of "") and then followed by local repo includes.